Library/Formula/libxmp-lite.rb

```diff
@@ -0,0 +1,10 @@
+class LibxmpLite < Formula
```

Comment:
Ugh... :frowning:.
Correct me if I misunderstand, but the "technical reason" is: any extant install of "coremod" would be effectively orphaned, i.e., since the coremod alias now always points to libxml-lite, there's no way to `brew rm coremod` on the old version, that command will always try to `brew rm libxmp-lite`.

My personal preference (in light of this) would be to revert back to "boneyarding" the coremod formula, while adding a new libxmp-lite formula. This actually seems a lot more prudent to me: given the fact that the libname has changed, "upgrading" coremod to libxmp-lite would probably cause breakage in anything dynamically linked to the old library. Plus, the formulae wouldn't conflict, since everything else has been renamed as well. Thoughts?